app/components/arcsi/Latest.vue

```diff
@@ -1,24 +1,44 @@
 <template>
   <div>
-    <h3 class="title-block">Arcsi's Latest</h3>
+    <h3 class="title-block">
+      <NuxtLink :to="`/archive/`">
+        Arcsi's Latest
+      </NuxtLink>
+    </h3>
     <div v-if="$fetchState.pending" class="flex flex-col items-center justify-center py-32">
       <img src="@/assets/img/preloader.svg" class="h-8 mb-2">
       <p>Loading...</p>
     </div>
-    <div v-else class="container relative py-10 latest-container">
-      <div v-swiper="swiperOption" class="relative" :loadtheme="false">
-        <div class="swiper-wrapper">
-          <div v-for="(episode, i) in arcsiEpisodesListSortedLatest" :key="episode + i" class="swiper-slide">
-            <ArcsiLatestBlock :episode="episode" :arcsilist="arcsiList" />
+    <div class="container relative py-10 latest-container" :class="{'opacity-0': $fetchState.pending} ">
+      <div ref="slider" class="arcsi-slider-wrapper">
+        <div ref="episodes" class="relative arcsi-episodes">
+          <div v-for="(episode, i) in arcsiEpisodesListSortedLatest" :key="episode + i">
+            <div class="episode-wrap" :style="{ 'width': episodeWidth + 'px' }">
+              <ArcsiLatestBlock :episode="episode" :arcsilist="arcsiList" />
+            </div>
           </div>
         </div>
       </div>
-      <div slot="button-prev" class="swiper-button-prev">
+      <a
+        v-show="sliderPosition > 0"
+        ref="button-prev"
+        href="#"
+        class="latest-nav previous"
+        :style="{ top: episodeWidth / 2 + 'px' }"
+        @click.prevent="previousBlock"
+      >
         <img src="@/assets/img/arrow-left.svg" alt="">
-      </div>
-      <div slot="button-next" class="swiper-button-next">
+      </a>
+      <a
+        v-show="sliderPosition < numberOfEpisodes - visibleEpisodes"
```

Comment:
What happens when there is only 1 more next episode? 

`sliderPosition = 8`
`numberOfEpisodes = 9`
`visibleEpisodes = 3`

Then `8 < (9-3)` doesn't hold.

Does that mean we can not reach the last one? 

Maybe I misunderstand which item is in focus, maybe it is never 3rd visible episode but the middle one. Still `7 < (9-3)` doesn't hold. 

I believe I am making some false assumptions on the calculations you are doing so forgive me if it is stupid to ask.

Reply:
@pvj the slider position starts from index 1, so 8 = 9
Nice catch about the last item resizement! I'll make a fix for it